README.md

```diff
@@ -227,45 +202,6 @@ Uncomment and modify the last section in case you're serializing classes with na
 In case you want to exclude serializable classes that are used, but never serialized at runtime,
 you will need to write custom rules with narrower [class specifications](https://www.guardsquare.com/manual/configuration/usage).
 
-<details>
-<summary>Example of custom rules</summary>
- 
-```proguard
--keepattributes RuntimeVisibleAnnotations,AnnotationDefault
-
-# kotlinx-serialization-json specific. Add this if you have java.lang.NoClassDefFoundError kotlinx.serialization.json.JsonObjectSerializer
--keepclassmembers class kotlinx.serialization.json.** {
-    *** Companion;
-}
--keepclasseswithmembers class kotlinx.serialization.json.** {
-    kotlinx.serialization.KSerializer serializer(...);
-}
-
-# Application rules
-
-# Change here com.yourcompany.yourpackage
--keepclassmembers @kotlinx.serialization.Serializable class com.yourcompany.yourpackage.** {
-    # lookup for plugin generated serializable classes
-    *** Companion;
-    # lookup for serializable objects
-    *** INSTANCE;
-    kotlinx.serialization.KSerializer serializer(...);
-}
-# lookup for plugin generated serializable classes
--if @kotlinx.serialization.Serializable class com.yourcompany.yourpackage.**
--keepclassmembers class com.yourcompany.yourpackage.<1>$Companion {
-    kotlinx.serialization.KSerializer serializer(...);
-}
-
-# Serialization supports named companions but for such classes it is necessary to add an additional rule.
-# This rule keeps serializer and serializable class from obfuscation. Therefore, it is recommended not to use wildcards in it, but to write rules for each such class.
--keepattributes InnerClasses # Needed for `getDeclaredClasses`.
--keep class com.yourcompany.yourpackage.SerializableClassWithNamedCompanion$$serializer {
-    *** INSTANCE;
-}
-```
-</details>
-
```

Comment:
Then perhaps we need to add this to the rules shipped with library

Reply:
@sandwwraith, create an additional file for these rules in the jar resources?

Reply:
I think so, yeah. Does it matter if it is an additional file or the same one?

Reply:
Unfortunately, it was not possible to reproduce the problem #2050 locally, so at the moment I don't think it's worth adding such built-in rules.
If necessary, users can always add them to their `proguard-rules.pro` file.

As soon as there is a small reproducer on which it will be possible to carefully learn the cause of the problems, then we can embed them into the jar.